src/components/layout/playground/applyForm.tsx

```diff
@@ -63,7 +63,13 @@ export const RequestDetails: React.FC<RequestProps> = ({ request }) => {
   }, [request]);
 
   const dueDateFormatted = useMemo(() => {
-    return DateTime.fromJSDate(request.dueDate).toFormat('MMMM dd, yyyy');
+    if (request.dueDate !== null) {
+      //If dueDate was set to a valid date, format it into a string
+      return DateTime.fromJSDate(request.dueDate).toFormat('MMMM dd, yyyy');
+    } else {
+      //Else a dueDate wasn't set for this request, so return null so the lack of a dueDate can be visualised
+      return null;
+    }
   }, [request]);
```

Comment:
To make it a bit smaller you could do:
`  const [timeUntilDue, isDue] = useMemo(
    () => (dueDate ? readableTimeDiff(dueDate) : [null, null]),
    [dueDate]
  );`
  But that won't work anyway, since readableTimeDiff is working activly with [null, null] as responses already, when the given timediff is now or in the past. So instead you can either use another value that won't appear natuarly in timeUntilDue like -1 or so, or you can introduce another variable which is just getting set whenever no dueDate is given like:
  `  const [timeUntilDue, isDue, hasNoDue] = useMemo(
    () => (dueDate ? readableTimeDiff(dueDate) : [null, null, true]),
    [dueDate]
  );` and check for that.
  Regarding the comments, the code should be self-explanatory, so no need for comments here